### DISCUSSION TITLE
Time to Kill

### DISCUSSION THREAD

**Discussion by DerivenLoki** [acemod/ACE-Anvil#97] [Q&A] [answered]
Arma 3 had a higher Time to Kill. Getting shot didnt immediately throw you unconscious, and a single bullet to the chest usually wasnt enough to put you out of the fight for a long period of time. Perhaps increasing the health of characters and increasing their shock resistance could help alleviate the fast paced feel of Reforger, and make encounters feel much more survivable

**Comment by Kexanone** [comment, marked as answer]
Don't open a new discussion for this. Make a comment in the actual feature request thread instead: https://github.com/acemod/ACE-Anvil/discussions/2
